resources/aries-test-b/testnet/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.7"
+services:
+  bootstrap-node:
+    image: ghcr.io/subspace/node:snapshot-2022-mar-09
+
+    volumes:
+      - /mnt/relaynet_test_volume/bootstrap-node:/var/subspace:rw
+    ports:
+      - "0.0.0.0:30333:30333"
+    restart: unless-stopped
+    command: [
+      "--chain", "testnet",
+      "--base-path", "/var/subspace",
+      "--wasm-execution", "compiled",
+      "--execution", "wasm",
+      "--pruning", "archive",
+      "--pool-kbytes", "51200",
+      "--telemetry-url", "wss://telemetry.polkadot.io/submit/ 1",
+      "--node-key", "f593e5994721e5c413d40cff0c5a230fdd86f7855197c5bf55230f710be6dbf7",
```

Comment:
This is a TERRIBLE idea, we use this secret key on farmnet and can't change it quickly. Why is it in public repo???